docs/getting-started/installing-service-binding.md

```diff
@@ -6,11 +6,182 @@ sidebar_position: 1
+
+- kubectl : Follow the [installation guide](https://kubernetes.io/docs/tasks/tools/) to install kubectl, a tool which runs command against Kubernetes cluster.
+
+## Install on Kubernetes
+
+1. Install Service Binding Operator
+
+```console
+$ kubectl apply -f https://github.com/redhat-developer/service-binding-operator/releases/download/v0.7.1/release.yaml
+
+namespace/service-binding-operator created
+configmap/service-binding-operator-manager-config created
+clusterrole.rbac.authorization.k8s.io/service-binding-operator created
+deployment.apps/service-binding-operator created
+clusterrole.rbac.authorization.k8s.io/service-binding-operator-servicebinding-viewer-role created
+customresourcedefinition.apiextensions.k8s.io/servicebindings.binding.operators.coreos.com created
+clusterrole.rbac.authorization.k8s.io/service-binding-operator-servicebinding-editor-role created
+clusterrolebinding.rbac.authorization.k8s.io/service-binding-operator created
+serviceaccount/service-binding-operator created
+```
+
+The operator will be installed in the "operators" namespace and will be usable from all other namespaces in the cluster.
+
+2. Check if operator has been installed successfully:
+
+```console
+$ kubectl get csv -n operators
+
+NAME                              DISPLAY                    VERSION   REPLACES   PHASE
+service-binding-operator.v0.4.0   Service Binding Operator   0.4.0                Succeeded
+```
+
+3. To verify the installation, create a Service Binding:
+
+```console
+$ kubectl apply -f - << EOD
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+        name: binding-request
+spec:
+        services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-demo
+EOD
+```
+
+The output should be:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"operators.coreos.com/v1alpha1","kind":"ServiceBinding","metadata":{"annotations":{},"name":"binding-request-backend","namespace":"default"},"spec":{"services":[{"group":"stable.example.com","kind":"Backend","name":"backend-demo","version":"v1"}]}}
+  creationTimestamp: "2021-05-17T07:09:48Z"
+  generation: 1
+  managedFields:
+  - apiVersion: operators.coreos.com/v1alpha1
+    fieldsType: FieldsV1
+    fieldsV1:
+      f:metadata:
+        f:annotations:
+          .: {}
+          f:kubectl.kubernetes.io/last-applied-configuration: {}
+      f:spec:
+        .: {}
+        f:services: {}
+    manager: kubectl
+    operation: Update
+    time: "2021-05-17T07:09:48Z"
+  - apiVersion: operators.coreos.com/v1alpha1
+    fieldsType: FieldsV1
+    fieldsV1:
+      f:spec:
```

Comment:
such big chunks of content are hard to maintain later in the document - especially if something gets changed on schema. Do we really expect that users will compare their output line by line with what we inject in the user doc?